NeoMathEngine/src/CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ target_sources(${PROJECT_NAME}
 add_library(NeoML::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
 
 target_compile_features(${PROJECT_NAME} PUBLIC cxx_std_11)
+set_target_properties( ${PROJECT_NAME} PROPERTIES CUDA_STANDARD 11 )
```

Comment:
https://github.com/neoml-lib/neoml/commit/76c2e33acd26289ab820d451d1930926744e8efc
Is this ok?

Reply:
Looks good. Thx.

Reply:
@SAngeliuk: And what about 1.0.76 release?